mantidproject/documents
# Minor update after v6.6 QA

During the Quality Assurance process for v6.6 release we decided we needed to update the files for both Manual and Smoke Testing. For more details see issue raised here - https://github.com/mantidproject/mantid/issues/35263

To Test
- Review the documentation changes
- Check the updates to the excel spreadsheets are consistent with changes needed (you need to be able to download and open excel spreadsheets to check this)
